chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
> I think it should be a question of consistency, purely to avoid debates about which declarator should have been used

That makes it sound like using one or the other is ambiguous, though. There's not any ambiguity if the rule is just "`var` for function scope, `let` for block scope". I'm not 100% wedded to that, but it's what we've done for years, and I'm just explaining that there's been a justification for it. (Places where `let` isn't used within blocks is generally old code.)

>  I'd argue that (1) is the right thing to do, but very well.

That could get annoying, though:

```javascript
var sql = "SELECT...";
Zotero.DB.query(sql);

sql = "INSERT..."
Zotero.DB.query(sql);

//
// more code
//

// Did I use 'sql' above? Do I need to go check? Or run without and wait for a strict mode warning?
var sql = "SELECT...";
Zotero.DB.query(sql);
```

And using different variables each time would be distracting.